azure-pipelines/publishing/github-release.yml

```diff
@@ -6,140 +6,183 @@ trigger:
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'ubuntu2204_arm64'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian10_arm32'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian10_arm64'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian10_x64'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian11_arm32'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian11_arm64'
-      imageVersion: ${{variables.containerImageVersion}}
-  - template: templates/release-docker-build-steps.yml
-    parameters:
-      targetOsArch: 'debian11_x64'
-      imageVersion: ${{variables.containerImageVersion}}
-
-- stage: release
-  condition: succeeded()
-  jobs:
-  - job: github_release
-    steps:
-    - task: DownloadBuildArtifacts@0
-      inputs:
-        buildType: 'current'
-        downloadType: 'specific'
-        downloadPath: '$(Build.ArtifactStagingDirectory)'
-
-    - task: CmdLine@2
-      inputs:
-        script: |
-          echo "Directory to be published: $(Build.ArtifactStagingDirectory)"
-          echo Content to be published:
-          ls -lR $(Build.ArtifactStagingDirectory)/**/*.tar
-      displayName: Release Information
-
-    - task: Bash@3
-      inputs:
-        targetType: 'inline'
-        script: 'sha256sum ./*.tar > ../hashes_sha256.txt'
-        workingDirectory: $(Build.ArtifactStagingDirectory)/github-release-artifacts
-      displayName: Compute hashes of archive files
-
-    - task: Bash@3
-      inputs:
-        targetType: 'inline'
-        script: 'cat ./hashes_sha256.txt'
-        workingDirectory: $(Build.ArtifactStagingDirectory)
-      displayName: Display hashes of archive files
-
-    - task: GitHubRelease@1
-      condition: eq(${{parameters.SkipPublishing}}, false)
-      inputs:
-        gitHubConnection: 'github_gauth'
-        repositoryName: 'microsoft/do-client'
-        action: 'create'
-        assets: |
-          $(Build.ArtifactStagingDirectory)/**/*.tar
-          $(Build.ArtifactStagingDirectory)/hashes*.txt
-        tagSource: 'userSpecifiedTag'
-        tag: '$(Release.Version)'
-        title: '$(Release.Title)'
-        isPreRelease: true
-        isDraft: ${{parameters.PublishAsDraft}}
-        changeLogCompareToRelease: 'lastNonDraftReleaseByTag'
-        changeLogCompareToReleaseTag: '$(Release.PreviousVersion)'
-        changeLogType: 'commitBased'
+- name: SkipPublishing
```

Comment:
>SkipPublishing [](http://example.com/codeflow?start=8&length=14)

is this where we will integrate the PMC publishing flow?

Reply:
We are currently attempting to create that as its own separate pipeline (release pipeline) but hitting some issues. We'll know more next week.